### PR TITLE
Update ci

### PR DESCRIPTION
**Pull request by bmagyar**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -10,21 +10,21 @@ on:
 
 jobs:
   # Re-enable once ros2_controls is released completely
```
Comment (Karsten1987) [inline .github/workflows/ci.yml:12]: can this comment be removed?

**Comment by Karsten1987** [comment]
Looking at the CI, it looks like the binary job is failing due to pluginlib using boost instead of std.

**Comment by destogl** [comment]
> Looking at the CI, it looks like the binary job is failing due to pluginlib using boost instead of std.

@bmagyar: you should add this line: [comment](https://github.com/ros-controls/ros2_control/pull/147#discussion_r499124758)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bmagyar) [reply .github/workflows/ci.yml]: gone

**Comment by bmagyar** [comment]
Merging this as it resolves at least part of the CI failures.

**Pull request merged**